src/pi-gen.ts

```diff
@@ -51,7 +51,10 @@ export class PiGen {
   }
 
   async build(verbose = false): Promise<exec.ExecOutput> {
-    const dockerOpts = this.getStagesAsDockerMounts()
+    let dockerOpts = this.getStagesAsDockerMounts()
+    if (this.config.dockerOpts !== undefined && this.config.dockerOpts !== '') {
```

Comment:
User's `dockerOpts` could actually also be an empty string (although rather pointless), so the second condition could be dropped.

Reply:
So you prefer to insert a space when it's empty string?

Reply:
No pref on my end, the env variable at least won't care. Just mentioned it, since the coverage check might complain for non-covered branch.